src/components/__tests__/ListItem.spec.tsx

```diff
@@ -44,9 +45,21 @@ describe('ListItem', () => {
       expect(getByText(title).instance.props.style[0].color).toBe(COLORS.text)
     })
   })
+    it('should handle press with locked exercise', async () => {
```

Comment:
This test is not testing it the press is correctly handled. So please rename it to "should show arrow disabled", and also you can remove everything from this test except the first three lines, as it does not make sense to 1) Check for title color, as this does not depend on the arrowDisabling (what is tested by this test) and 2) not trigger an action and then check for the same stuff again.

Reply:
I have removed this test already..
should I try writing another test?

Reply:
no its fine, like it is.